docs/team/jingyenloh.md

```diff
@@ -3,44 +3,46 @@ layout: page
+## Project: ResiReg
 
-AddressBook - Level 3 is a desktop address book application used for teaching Software Engineering principles. The user interacts with it using a CLI, and it has a GUI created with JavaFX. It is written in Java, and has about 10 kLoC.
+**ResiReg** is a desktop residential college management application used for teaching Software Engineering principles. The user interacts with it using a CLI, and it has a GUI created with JavaFX. It is written in Java, and has about 20 kLoC.
 
 Given below are my contributions to the project.
 
-* **New Feature**: Added the ability to undo/redo previous commands.
-  * What it does: allows the user to undo all previous commands one at a time. Preceding undo commands can be reversed by using the redo command.
-  * Justification: This feature improves the product significantly because a user can make mistakes in commands and the app should provide a convenient way to rectify them.
-  * Highlights: This enhancement affects existing commands and commands to be added in future. It required an in-depth analysis of design alternatives. The implementation too was challenging as it required changes to existing commands.
-  * Credits: *{mention here if you reused any code/ideas from elsewhere or if a third-party library is heavily used in the feature so that a reader can make a more accurate judgement of how much effort went into the feature}*
+* **New Feature**: Added the ability to archive the current Semester.
+  * What it does: allows the user to move the current Semester's allocation into an archived file. The application is then set to the next Semester, and future allocations will be set with respect to that Semester.
+  * Justification: This feature improves the product significantly because the allocation data the user operates on is semester specific. This allows them to conveniently refresh the data for a new Semester with a single command, while ensuring all data from the previous semester is preserved.
+  * Highlights: This enhancement affects existing allocations and models to be added in future, such as Bills. It required an in-depth analysis of design alternatives. The implementation too was challenging as it required the Semester to be made Observable, rather than a POJO like the rest of the AddressBook/ResiReg models.
 
-* **New Feature**: Added a history command that allows the user to navigate to previous commands using up/down keys.
+* **New Feature**: Added `--allocated` and `--vacant` flags to the `rooms` command
+  * What it does: allows the user to optionally view only allocated or vacant rooms, versus the original command where it would display all the rooms.
+  * Justification: This allows the user more find grained viewing of the rooms, for when they only want to view rooms with students residing in them (e.g. for maintainence services), or when they only want to view vacant rooms (e.g. to see if last minute allocations are allowed for an Exchange student)
```

Comment:
```suggestion
  * Justification: This allows the user more fine-grained viewing of the rooms, for when they only want to view rooms with students residing in them (e.g. for maintainence services), or when they only want to view vacant rooms (e.g. to see if last minute allocations are allowed for an Exchange student)
```
Can you proof-read your changes first before PR-ing? Please don't waste your reviewers' time.